core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -796,8 +796,14 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
   /**
    * Returns the effect resulting from mapping the success of this effect to unit.
    */
+  @deprecated("use unit", "1.0.0")
```

Comment:
Can you make this one delegate to the other one, to avoid duplicated code?

Reply:
Removed duplicate code by delegating to new function.